src/components/pagination.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function getPageRange(size) {
+  return [...Array(size).keys()].map((i) => i + 1);
+}
+
+class Pagination extends React.Component {
+  static propTypes = {
+    page: PropTypes.number,
+    pages: PropTypes.number,
+    onPageChange: PropTypes.func
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
No need to use state here.

Reply:
That makes sense. Using the url as the source of truth would be helpful for sharing though.

Reply:
Agreed. The URL should be the source of truth.